docs/content/_fault_tree/pubsub-consumers.md

```diff
@@ -0,0 +1,8 @@
+---
+components: Heroic consumers using PubSub for ingestion, and Bigtable and Elasticsearch for storage
+---
+
+Some assumptions are made that negatively affect the accuracy of the model and the calculated probabilities should be seen as being quite pessimistic. The biggest assumptions are:
+
+- Numbers for Google Cloud services are taken from their SLAs. In reality, the services will typically be much more reliable than the numbers listed since SLAs are contractual agreements and skew conservatively.
```

Comment:
Can we link out to where all the probabilities were taken for future reference? I thought I remember seeing them somewhere.

Reply:
The links to the SLAs are in the mef file, do you think I should also add them to the doc page?

Reply:
ah thats where I saw them which wasn't part of this commit. no thats fine having them in there.